BlackNectar/BlackNectar/SearchFilterViewController.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  MapViewSearchFilter.swift
+//  BlackNectar
+//
+//  Created by Cordero Hernandez on 2/6/17.
+//  Copyright © 2017 BlackSource. All rights reserved.
+//
+
+import Archeota
+import AromaSwiftClient
+import CoreLocation
+import Foundation
+import Kingfisher
+import MapKit
+import UIKit
+
+class SearchFilterViewController: UITableViewController, MKMapViewDelegate, CLLocationManagerDelegate {
+    
+    @IBOutlet weak var mapView: MKMapView!
+    
+    var currentCoordinates: CLLocationCoordinate2D?
+    
+    var stores: [Store] = []
+    
+    var selectedPin: MKPlacemark?
+    var distance = 0.0
+    
+    var showFarmersMarkets = true
+    var showGroceryStores = true
+    
+    typealias callBack = ([Store]) -> ()
```

Comment:
This typealias should only have to be defined once throughout the project. It can then be used throughout.